src/graphql/main/schema.graphql

```diff
@@ -259,6 +259,16 @@ type Coordinates {
   longitude: Float!
 }
 
+type Country {
+  id: CountryCode!
```

Comment:
I think `id` need to be of type `ID` for the cache to work.

but you can maybe verify if that's the case.

useful reference: https://www.apollographql.com/docs/react/caching/overview#2-generate-cache-ids

Reply:
Per ChatGPT it shouldn't make any difference, I also didn't see any mention of it in that documentation. Either way our current behavior is to cache the entire Globals object.